src/SFML/Graphics/Texture.cpp

```diff
@@ -453,8 +453,8 @@ void Texture::update(const std::uint8_t* pixels)
 ////////////////////////////////////////////////////////////
 void Texture::update(const std::uint8_t* pixels, const Vector2u& size, const Vector2u& dest)
 {
-    assert(dest.x + size.x <= m_size.x);
-    assert(dest.y + size.y <= m_size.y);
+    assert(dest.x + size.x <= m_size.x && "Destination x coordinate is outside of texture");
+    assert(dest.y + size.y <= m_size.y && "Destination y coordinate is outside of texture");
```

Comment:
Non-native speaker, can one really say "outside of texture" instead of "outside of the texture"?

Reply:
It's not something you'd say verbally but in code it's common to drop "the" for the sake of brevity.